ltp-production.yaml

```diff
@@ -1723,10 +1723,11 @@ projects:
     url: https://bugs.linaro.org/show_bug.cgi?id=5806
     active: true
     intermittent: true
-  - environments: *environments_qemu
+  - environments: *environments_all
```

Comment:
isn't we disabling this from a bit too much now?
if it only fails on qemu and compat?

Reply:
Test case is failing on one of those environments is expected to fail on future devices / environments. It is not easy to maintain the list of environments update manually.

Reports are getting noise due to this test case added to limited number of environments as a known issue.